app/src/components/nav-menu/Friends.js

```diff
@@ -1,12 +1,62 @@
 import React, {Component} from 'react';
+import {NavLink} from 'react-router-dom';
+import NavigateMenu from './NavigateMenu';
+import FriendsRequests from "../../common/FriendsRequests";
+import UsersDataRequests from "../../common/UsersDataRequests";
+import endPointsList from "../../common/endPointsList";
 
-class Friends extends Component{
+class Friends extends Component {
 
-    constructor(props){
+    constructor(props) {
         super(props);
+        this.state = {
+            friendsInfo: {}
+        };
+
+        this.loadFriendsDataFromDb();
+    }
+
+    async loadFriendsDataFromDb() {
+        const userInfo = await UsersDataRequests.loadUserInfo();
+        const friendsInfo = await FriendsRequests.loadFriendsData(userInfo.friendsList);
+        this.setState({...this.state, userInfo, friendsInfo});
```

Comment:
Why do you store data in the state? You have to store all the data in the redux store.